### PR TITLE
View chart

### PR DESCRIPTION
**Pull request by jing-xiang**
- Add view chart feature
- Add JUnit tests

### CODE REVIEW

File: src/test/java/longah/util/ChartTest.java
Comment (1simjustin) [inline src/test/java/longah/util/ChartTest.java:1]: Can shift all these commented test code to an unused code folder? Course site has guidelines for this

File: src/main/java/longah/commands/ViewCommand.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            memberNames.add(member.getName());
+            memberBalances.add(member.getBalance());
+        }
+        try {
+            if (this.taskExpression.equals("chart")) {
+                Logging.logInfo("Loading balances chart...");
+                Chart.viewBalancesBarChart(memberNames, memberBalances);
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```
Comment (1simjustin) [inline src/main/java/longah/commands/ViewCommand.java:51]: Can still throw as LongAhException, it still has functionality to take in a string message to output it. I'm worried try catch in main won't catch this

File: src/main/java/longah/commands/ViewCommand.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        }
+        if (memberList.size() == 0) {
+            throw new LongAhException("No members to display");
+        }
+        for (Member member : memberList) {
+            memberNames.add(member.getName());
+            memberBalances.add(member.getBalance());
+        }
+        try {
+            if (this.taskExpression.equals("chart")) {
```
Comment (1simjustin) [inline src/main/java/longah/commands/ViewCommand.java:46]: This part can be done as a prominent happy path

File: src/main/java/longah/commands/ViewCommand.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    public ViewCommand(String commandString, String taskExpression) {
+        super(commandString, taskExpression);
+    }
+
+    /**
+     * Executes the view chart command.
+     *
+     * @param group The group to execute the command on.
+     */
+    public void execute(Group group) throws LongAhException {
```
Comment (1simjustin) [inline src/main/java/longah/commands/ViewCommand.java:30]: Is there a need for dual command syntax for this?

**Review by 1simjustin — APPROVED** [review]
Generally LGTM

File: src/main/java/longah/commands/ViewCommand.java (hunk 1 above)
Comment (jing-xiang) [reply src/main/java/longah/commands/ViewCommand.java:51]: image

Resolved

File: src/test/java/longah/util/ChartTest.java
Comment (jing-xiang) [reply src/test/java/longah/util/ChartTest.java:1]: resolved

**Pull request merged**